### PR TITLE
Fix EsLint failures

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose
$subject

### Related Issues
- None

### Related PRs
- https://github.com/wso2/identity-apps/pull/3521

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx:2]: Can we not change the year?
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx [hunk 2]
```diff
@@ -62,9 +62,9 @@ interface AttributeListItemPropInterface extends TestableComponentInterface {
 /**
  * Selected Attribute list item component.
  *
- * @param {AttributeListItemPropInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
- * @return {React.ReactElement}
+ * @returns
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx:67]: Can we mention what we are returning?

File: apps/console/src/features/applications/components/settings/attribute-management/index.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/attribute-management/index.ts:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/oidc-scopes/models/oidc-scopes.ts [hunk 4]
```diff
@@ -1,16 +1,16 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline apps/console/src/features/oidc-scopes/models/oidc-scopes.ts:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx [hunk 6]
```diff
@@ -117,9 +117,9 @@ export interface StrictSegmentedAccordionTitleActionInterface {
 /**
  * Segmented accordion title component.
  *
- * @param {SegmentedAccordionTitlePropsInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
- * @return {ReactElement}
+ * @returns
```
Comment (dasuni-30) [inline modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx:122]: Can we add what we are returning?

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx [hunk 7]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx (hunk 2 above)
Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx]: resolved by https://github.com/wso2/identity-apps/pull/3522/commits/a643633e78865647ca46a73007235365cd6ababf

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx (hunk 6 above)
Comment (mpmadhavig) [reply modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx:122]: resolved with https://github.com/wso2/identity-apps/pull/3522/commits/59a9f310bf60598bccd0731fa9aa146bf7f412cc

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**